src/ume_mpi.cc

```diff
@@ -92,16 +134,24 @@ int main(int argc, char *argv[]) {
 
   VEC3V_T pgrad, zgrad;
   Ume::Timer orig_time;
-  Ume::gradzatz(mesh, zfield, zgrad, pgrad);
+  Ume::gradzatz(mesh, zfield, zgrad, pgrad, 0);
```

Comment:
```suggestion
  Ume::gradzatz(mesh, zfield, zgrad, pgrad);
```

No need for the extra argument (I think this was needed for the caliper branch). Same for the other calls to the other routines.

Reply:
My bad, corrected it.